components/custom-button.tsx

```diff
@@ -0,0 +1,17 @@
+import { blueDarkAlt } from '@pxblue/react-native-themes';
+import React from 'react';
+import { Button, useTheme } from 'react-native-paper';
+
+export const MyCustomButton: typeof Button = (props) => {
+    const theme = useTheme(props.theme);
+    return (
+        <Button
+            {...props}
+            theme={
+                props.mode === 'contained' && theme.dark
+                    ? Object.assign({}, JSON.parse(JSON.stringify(blueDarkAlt)), props.theme)
```

Comment:
I'm not a huge fan of the stringify/parse approach here. It has some limitations with things like functions, undefineds, etc. I would almost prefer to just bring in lodash.clonedeep as a dependency and use that.

https://www.npmjs.com/package/lodash.clonedeep